phpstan.neon

```diff
@@ -0,0 +1,11 @@
+includes:
+	- vendor/phpstan/phpstan-strict-rules/rules.neon
```

Comment:
This should also include `extension.neon`.

Reply:
Should it? I mean - this file is only for build testing, I could move it to `tests/` folder to clarify its purpose.